### PR TITLE
Country Name changes

### PR DESCRIPTION
**Pull request by MaiaTechnology**
Now we can get the company name when you select any dialing code from the dropdown.

### CODE REVIEW

File: src/VuePhoneNumberInput/index.vue [hunk 1]
```diff
@@ -117,7 +120,10 @@
         const countryAvailable = isCountryAvailable(locale)
 
         if (countryAvailable && locale) {
-          this.countryCode = locale
+          this.countryCode = locale;
+          this.countryName = this.defaultCountryName || countries.find(function(element) {
```
Comment (aawlee) [inline src/VuePhoneNumberInput/index.vue]: We're doing 2 passes through the countries list - once in isCountryAvailable(), and another in the .find(). I'd suggest changing isCountryAvailable() to findCountry() and use the returned value instead

File: src/App.vue [hunk 2]
```diff
@@ -106,14 +125,16 @@
...
+            :error="hasErrorActive"
             @update="onUpdate2"
           />
         </div>
         <br>
         <div class="component">
           <h3>Datas returned with "onUpdate" event</h3>
           <hr>
-          <b>v-model</b> : {{ phoneNumber }}
+          <b>v-model</b> : {{ phoneNumber }}agirdhar
```
Comment (aawlee) [inline src/App.vue]: Sneaky, leaving your tag in random places? ;)

File: src/App.vue [hunk 3]
```diff
@@ -157,6 +178,7 @@
         phoneNumber: '0665656565',
         phoneNumber2: null,
         defaultCountry: 'FR',
+        defaultCountryName: '',
```
Comment (aawlee) [inline src/App.vue]: Set the default to the name for FR

File: src/App.vue (hunk 2 above)
Comment (MaiaTechnology) [reply src/App.vue]: I don't know, how my name is all around. Let me fix this

File: src/VuePhoneNumberInput/index.vue (hunk 1 above)
Comment (MaiaTechnology) [reply src/VuePhoneNumberInput/index.vue]: Nice suggestion, Ideally I have to do it like that. I was not preferring not to disturb the main functionality. I am completely agree on your suggestion so I will implement the same.

Thanks
Ashish

File: src/App.vue (hunk 3 above)
Comment (MaiaTechnology) [reply src/App.vue]: Sure, missed in the last test case testing

File: src/VuePhoneNumberInput/index.vue [hunk 4]
```diff
@@ -114,22 +117,25 @@
       },
       locale () {
         const locale = this.defaultCountryCode || (!this.noUseBrowserLocale ? browserLocale() : null)
-        const countryAvailable = isCountryAvailable(locale)
+        const countryNameByLocale = countries.find(function(element) {
+          return element.iso2 === locale;
+        }).name;
```
Comment (aawlee) [inline src/VuePhoneNumberInput/index.vue]: could run into null here. better to store the country on its own and check for null as an equivalent of isCountryAvailable(), then get the name after if it exists

Comment (MaiaTechnology) [reply src/VuePhoneNumberInput/index.vue]: Thanks for pointing this out. Implement the changes

File: src/VuePhoneNumberInput/index.vue [hunk 5]
```diff
@@ -114,10 +117,13 @@
...
         const locale = this.defaultCountryCode || (!this.noUseBrowserLocale ? browserLocale() : null)
-        const countryAvailable = isCountryAvailable(locale)
+        const countryAvailable = countries.find(function(element) {
+          return element.iso2 === locale;
+        });
 
         if (countryAvailable && locale) {
-          this.countryCode = locale
+          this.countryCode = locale;
+          this.countryName = this.defaultCountryName || countryAvailable.name;
```
Comment (aawlee) [inline src/VuePhoneNumberInput/index.vue]: Here should just be this.countryName = countryAvailable.name;
Can put the this.countryName = this.defaultCountryName in the else if

File: src/VuePhoneNumberInput/index.vue [hunk 6]
```diff
@@ -114,10 +117,13 @@
       },
       locale () {
         const locale = this.defaultCountryCode || (!this.noUseBrowserLocale ? browserLocale() : null)
-        const countryAvailable = isCountryAvailable(locale)
+        const countryAvailable = countries.find(function(element) {
```
Comment (aawlee) [inline src/VuePhoneNumberInput/index.vue]: Minor nitpicking, I'd rename the variable to just "country"

**Comment by ashishgirdhar** [comment]
@aaronwittlee - I have implement the changes, Please check

**Comment by aawlee** [comment]
Nice @ashishgirdhar, looks good.

**Pull request merged**